agent/structs/acltemplatedpolicy/policies/ce/nomad-server.hcl

```diff
@@ -1,10 +1,12 @@
 
-acl = "write"
+acl  = "write"
+mesh = "write"
```

Comment:
Why do Nomad servers need the `mesh:write` permission?

Reply:
from https://developer.hashicorp.com/nomad/docs/integrations/consul/acl#nomad-agents
> Nomad servers also create [configuration entries](https://developer.hashicorp.com/consul/docs/connect/config-entries) for Consul Service Mesh, so the specific permissions vary slightly between Nomad servers and clients.